src/to-car.ts

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs'
+import { Readable, Writable } from 'stream'
+
+import { CarWriter } from '@ipld/car'
+import { importer } from 'ipfs-unixfs-importer'
+import { UnixFSEntry } from 'ipfs-unixfs-exporter'
+
+import { CID } from 'multiformats'
+import all from 'it-all'
+import pipe from 'it-pipe'
+
+import { sha256 } from 'multiformats/hashes/sha2'
+
+import normalizeAddInput from 'ipfs-core-utils/src/files/normalise-input/index'
+
+import globSource from 'ipfs-utils/src/files/glob-source'
+import { WriterChannel } from '@ipld/car/lib/writer'
+
+export async function packFileToCarFs ({ input, output }: { input: string | Iterable<string> | AsyncIterable<string>, output?: string }) {
+  const writable = fs.createWriteStream(output || 'output.car') // TODO: How to handle default naming given we do not have cid
```

Comment:
how about a _write to tmp then mv to output location dance_. 💃 
- write it with a random name to the os tmp dir
- grab the root CID
- mv to the cwd with the root cid as the name

it might be nicer to grab the dir or file name from the input and just add `.car` on the end, but for cases like mutiple files that get wrapped with a dir it would have to be the cid or something clever like `filename[0]+3.car` which might be too fancy.

Reply:
> how about a write to tmp then mv to output location dance. 💃

This seems a great idea!

> it might be nicer to grab the dir or file name from the input and just add .car on the end

👌🏼 

Since we are not supporting multiple files in one operation at the moment (unless a directory), we can worry with that later on

Reply:
Please check the new commit. I also needed to take into account when receiving inputs that are not a string. On these cases I rename to the root cid. So I also changed `packFileToCar` to return the root cid